apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/post-published-sharing-modal/index.tsx

```diff
@@ -0,0 +1,253 @@
+import { Modal, Button } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect, useRef, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon, globe, link as linkIcon } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import classnames from 'classnames';
+import React from 'react';
+import ClipboardButton from 'calypso/components/forms/clipboard-button';
+import FormInputCheckbox from 'calypso/components/forms/form-checkbox';
+import FormLabel from 'calypso/components/forms/form-label';
+import SocialLogo from 'calypso/components/social-logo';
+import { selectors as wpcomWelcomeGuideSelectors } from '../store';
+import postPublishedImage from './images/illo-share.svg';
+import useSharingModalDismissed from './use-sharing-modal-dismissed';
+import type { SelectFromMap } from '@automattic/data-stores';
+import './style.scss';
+
+type WpcomWelcomeGuideSelectors = SelectFromMap< typeof wpcomWelcomeGuideSelectors >;
+type CoreEditorPlaceholder = {
+	getCurrentPost: ( ...args: unknown[] ) => { link: string; title: string };
+	getCurrentPostType: ( ...args: unknown[] ) => string;
+	isCurrentPostPublished: ( ...args: unknown[] ) => boolean;
+};
+const FB_APP_ID = '249643311490';
+
+const PostPublishedSharingModal: React.FC = () => {
+	const isDismissedDefault = window?.sharingModalOptions?.isDismissed || false;
+
+	const { isDismissed, updateIsDismissed } = useSharingModalDismissed( isDismissedDefault );
+
+	const { link, title } = useSelect(
+		( select ) => ( select( 'core/editor' ) as CoreEditorPlaceholder ).getCurrentPost(),
+		[]
+	);
+	const postType = useSelect(
+		( select ) => ( select( 'core/editor' ) as CoreEditorPlaceholder ).getCurrentPostType(),
+		[]
+	);
+
+	const isCurrentPostPublished = useSelect(
+		( select ) => ( select( 'core/editor' ) as CoreEditorPlaceholder ).isCurrentPostPublished(),
+		[]
+	);
+	const previousIsCurrentPostPublished = useRef( isCurrentPostPublished );
+	const shouldShowFirstPostPublishedModal = useSelect(
+		( select ) =>
+			(
+				select( 'automattic/wpcom-welcome-guide' ) as WpcomWelcomeGuideSelectors
+			 ).getShouldShowFirstPostPublishedModal(),
+		[]
+	);
+
+	const [ isOpen, setIsOpen ] = useState( false );
```

Comment:
We'll need to make sure to remove this line, right?

Reply:
Or is it fine because it's set to false?

Reply:
yea false is correct